bowling/src/BowlingLane.cpp

```diff
@@ -0,0 +1,29 @@
+#include "BowlingLane.hpp"
+
+BowlingLane::GameState BowlingLane::checkGameState() {
+    for (const auto& player : players_) {
```

Comment:
There is a loop over players but you always return after the first player check. What if the last player has not finished yet while the first has? Add a proper test case.